yarn.lock

```diff
@@ -7917,6 +7917,7 @@ which@^2.0.1:
     isexe "^2.0.0"
 
 "wrap-ansi-cjs@npm:wrap-ansi@^7.0.0", wrap-ansi@^7.0.0:
+  name wrap-ansi-cjs
```

Comment:
@xyloflake What's this for?

Reply:
IDK it got added by it's own

Reply:
I've removed it.

Reply:
@xyloflake I think that's related to Vite downgrade commit.
Please revert this as long as there's no need for downgrade.

Reply:
@xyloflake You can actually drop all `Update dependencies` commits as they don't belong here.